examples/linear_model/plot_sgd_loss_functions.py

```diff
@@ -38,5 +38,5 @@ def modified_huber_loss(y_true, y_pred):
 plt.ylim((0, 8))
 plt.legend(loc="upper right")
 plt.xlabel(r"Decision function $f(x)$")
-plt.ylabel("$L(y, f(x))$")
+plt.ylabel("$L(y = 1, f(x))$")
```

Comment:
@amueller suggested `L(y=1, ...` and I don't really see how the spaces increase clarity.